app/assets/javascripts/ama_layout/autocorrect_email.js

```diff
@@ -1,40 +1,45 @@
-var domains = ['msn.com', 'bellsouth.net', 'telus.net', 'comcast.net', 'optusnet.com.au', 'earthlink.net', 'qq.com',
+window.onload = function(){
+  var domains = ['msn.com', 'bellsouth.net', 'telus.net', 'comcast.net', 'optusnet.com.au', 'earthlink.net', 'qq.com',
                 'sky.com', 'icloud.com', 'mac.com', 'sympatico.ca', 'googlemail.com', 'att.net', 'xtra.co.nz', 'web.de',
                 'cox.net', 'gmail.com', 'ymail.com', 'aim.com', 'rogers.com', 'verizon.net', 'rocketmail.com',
                 'google.com', 'optonline.net', 'sbcglobal.net', 'aol.com', 'me.com', 'btinternet.com', 'charter.net',
                 'shaw.ca', 'xplore.com', 'abnorth.com', 'pentnet.net', 'canadasurf.net', "ama.ab.ca"];
 
-var secondLevelDomains = ["yahoo", "hotmail", "mail", "live", "outlook"]
-
-var $email = $('[type="email"]');
-var $hint = $("*.email_hint");
-
-$email.on('blur', function() {
-  $(this).mailcheck({
-    domains: domains,
-    secondLevelDomains: secondLevelDomains,
-    suggested: function(element, suggestion) {
-      var text = "Did you mean <span class='suggestion'>" +
-        "<span class='address'>" + suggestion.address + "</span>" +
-        "@<a href='#' class='email_domain'>" + suggestion.domain +
-        "</a></span>?";
-
-      element.nextAll(".email_hint:first").html(text).fadeIn(150);
-    },
-    empty: function(element) {
-      element.nextAll(".email_hint:first").html("");
-    }
+  var secondLevelDomains = ["yahoo", "hotmail", "mail", "live", "outlook"]
+
+  var $email = $('[type="email"]');
+
+  $email.on('blur', function() {
+    $(this).mailcheck({
+      domains: domains,
+      secondLevelDomains: secondLevelDomains,
+      suggested: function(element, suggestion) {
+        var text = "Did you mean <span class='suggestion'>" +
```

Comment:
can you use create element instead of text markip
https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement
https://developer.mozilla.org/en-US/docs/Web/API/Element